packages/astro/test/units/vite-plugin-astro-server/request.test.js

```diff
@@ -51,6 +51,10 @@ describe('vite-plugin-astro-server', () => {
 				logging
 			);
 
+			// `handleRequest` below has a call to telemetry when an error is found. Telemetry should already be disabled
+			// in CI by default, but we'll make sure to disable it here too just in case.
+			process.env.ASTRO_TELEMETRY_DISABLED = true;
```

Comment:
I don't like that we now need to remember to do this in every test that does dev server requests. That would mean all of these tests need it too: https://github.com/withastro/astro/tree/main/packages/astro/test/units/dev Plus any fixture-based tests that run dev.

What I would probably suggest doing is making this an option on the `Environment` object and then having the tests pass through `false`.

Reply:
That's understandable, will do!